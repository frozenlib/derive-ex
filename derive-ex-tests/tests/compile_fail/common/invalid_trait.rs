use derive_ex::derive_ex;

#[derive_ex(Unknown)]
struct X;

fn main() {}
