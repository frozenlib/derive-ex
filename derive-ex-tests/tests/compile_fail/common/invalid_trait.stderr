error: unsupported trait
 --> tests/compile_fail/common/invalid_trait.rs:3:13
  |
3 | #[derive_ex(Unknown)]
  |             ^^^^^^^
